_posts/2018-05-24-Spring-IO-2018.md

```diff
@@ -0,0 +1,777 @@
+
+But how can we define Machine Learning? [Andrew Ng](https://twitter.com/AndrewYNg){:target="_blank"}, Co-founder of Coursera and Adjunct Professor of Stanford University defined Machine learning in his introduction course "Welcome To Machine Learning"  <sup>1</sup> as "*Machine Learning is the science of getting computers to learn, without being explicitly programmed*". An example that Andrew gave was a cleaning robot that can tidy your house. Instead you program the algorithm explicitly on how it should clean. You can for instance let the robot watch you while you demonstrate the tasks on how it should clean and learn from it.
+
+Later on he gave examples of different categories of machine learning.
+
+### Categories of Machine Learning
+
+#### Supervised learning
+
+This was the category where James gave the most examples of during his talk.
+Supervised learning is where you train your model with a dataset which contains the initial data and its correct answers.
+The more training data you have, the more accurate your predictions will be.
+
+##### Regression example
+
+An example he showed us was the prediction of housing prices using regression.
+
+<span class="image center"><img class="p-image" alt="House price prediction (From Andrew Ng Learning course)" src="/img/spring-io-2018/houseprice.png" class="image fit" style="max-height:'476px'; max-width='876px'">
+<small class="center">(From Andrew NG's Machine learning course)</small></span>
+
+In this example, the dataset consists of instances with a square footage (input) and price (output).
+With a regression, we can predict a continuous valued price.
+
+##### Classification example
+
+<div class="row"><span class="image left"><img alt="Iris flower classification using machine learning" src="/img/spring-io-2018/iris-dataset.png" class="image fit" style="max-width='374.5px'; max-height='374.5px;'">
+<small>Source:<a href="https://en.wikipedia.org/wiki/User:Nicoguaro/Gallery">Nicoguaro's Wikipedia media gallery</a> (CC BY 4.0) An example of Supervised Learning using classification </small></span>
+<span class="image right"><img alt="Iris flower classification using machine learning" src="/img/spring-io-2018/test-data.png" cslass="image fit">
+<small></small></span></div>
+
+Another example of Supervised Learning is to determine a certain species of an Iris flower.
+The algorithm tries to determine the species of the flower with the Sepal and Petal size as input.
+
+#### Unsupervised Learning
+
+For Unsupervised learning on the other hand, you don't give the right answers with your dataset.
+Your learning algorithm will try to find a structure in the given data.
+
+A method to try to find a structure, is to do it by clustering.
+This means the data is 'grouped' in clusters together with data that more or less belongs to each other.
```

Comment:
is 'grouped' together in clusters (put 'together' next to 'grouped')

to each other ==> 'together'